js_modules/dagster-ui/packages/ui-components/src/components/__stories__/BaseButton.stories.tsx

```diff
@@ -31,35 +32,35 @@ export const Default = () => {
 export const Fill = () => {
   return (
     <Group direction="column" spacing={8}>
-      <BaseButton label="Button" fillColor={CoreColors.Gray900} textColor={CoreColors.White} />
+      <BaseButton label="Button" fillColor={Colors.accentGray()} textColor={Colors.alwaysWhite()} />
```

Comment:
why is this `alwaysWhite`?

Reply:
We're currently using white in all cases for the text on toasters and filled buttons, regardless of theme, but we were previously using `CoreColors.White` directly to accomplish this. I added `alwaysWhite()` within this PR to make it a "themable" color function like the others, though we could come up with a more appropriate semantic name for it.